examples/plot_multi_axes.py

```diff
@@ -0,0 +1,111 @@
+"""
+Test Matplotlib Figures with SubPlots Using Matplotcheck
+========================================================================
+
+Matplotlib figures can have more than one subplot. In this vignette, you will 
```

Comment:
It's probably worth adding a code section that shows the anatomy of a matplotlib plot. you can just create an empty plot with a figure title and then subplot titles

fig, (ax1, ax2) = plt.subplots(1, 2, figsize = (10, 6))
# add suptitle for the entire figure: 
# att titles for each subplot
# add x and y labels for each subplot

Reply:
i implemented this